src/main/archetype/all/pom.xml

```diff
@@ -88,6 +88,12 @@
                             <artifactId>magento-graphql</artifactId>
                             <filter>true</filter>
                         </embedded>
+                        <embedded>
+                            <groupId>com.adobe.cq</groupId>
+                            <artifactId>core.wcm.components.core</artifactId>
```

Comment:
Should embed the bundle via ui.apps pom.xml, same like AEM archetype?

Reply:
We can, but everything is currently embedded in the `all` package so I kept doing what was already done ... the `ui.apps` actually doesn't embed any bundle.